src/App.tsx

```diff
@@ -15,6 +16,7 @@ export default function App() {
           <Routes>
             <Route path="/" element={<Home />} />
             <Route path="/register" element={<Register />} />
+            <Route path="/login" element={<Login />} />
```

Comment:
The extra route isn't necessary, the Home component already contains the Login component.